### ISSUE TITLE
Convert `update_data.py` to a function and call it from `cli/query.py`

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Data#159]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue would be to convert over the functionality of [update_data.py](https://github.com/scribe-org/Scribe-Data/blob/main/src/scribe_data/wikidata/update_data.py) into a function that can then be called in the CLI query process.

Generally the steps for this would be:
- Put all of [update_data.py](https://github.com/scribe-org/Scribe-Data/blob/main/src/scribe_data/wikidata/update_data.py) into a function
- This function would have `language` and `data_type` (new `word_type`) arguments
- We'd pass the arguments above from the CLI to the function
- Delete the data_update.txt and the date_table.txt files
- Remove the code that generates them from the new function

At the end of this issue:
```bash
# Rather than:
python src/scribe_data/wikidata/update_data.py
python src/scribe_data/wikidata/update_data.py ["German"] ["nouns"]
# We'd have:
scribe-data q -all
scribe-data q -lang German -dt nouns
```

### Contribution

@mhmohona will work on this as a part of GSoC 2024 ☀️ Please write in here so we can assign, and let us know if there are any questions!

**Comment by mhmohona** [comment]
Will work on this issue.

**Comment by andrewtavis** [comment]
Closed by #161 🙌 Amazing work here, @mhmohona! 😊😊
